### PR TITLE
LPS-160082 | Add supported query params to Headless API Explorer for individual elements

### PR DESCRIPTION
**Pull request by Hijky**
**Background**
Add supported query params to Headless API Explorer for individual elements

**Test Plan**
Given a document of a site
When with curl I request getDocument with fields=id
Then in a response I received an id value only 

Given a document of a site
When with curl I request getDocument with restrictFields equal all fields except title field
Then in a response I receive a body with title field values only

Given a content structure created
And Given web content created
When with curl I request getStructuredContentByVersion with fields=id
Then in a response I received an id value only

Given a content structure created
And Given web content created
When with curl I request getStructuredContentByVersion with restrictFields equal all fields except title field
Then in a response I receive a body with title field values only

Given an AssetLibrary with keyword created
When with curl I request getKeyword with fields=name
Then in a response I receive a body with name values only

Given an AssetLibrary with keyword created
When with curl I request getKeyword with restrictFields equal all fields except id field
Then in a response I receive a body with id field values only

**JIRA Link:**
https://issues.liferay.com/browse/LPS-160082

### CODE REVIEW

**Comment by Hijky** [comment]
ci:test:headless-frontend-functional:nocompile

**Comment by Hijky** [comment]
@john-co It seems the failed not related to my PR.

**Comment by john-co** [comment]
Just started reviewing :)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessAdminContentAPI/SupportContentVersioning/HeadlessAdminContentSupprotDifferentParameters.testcase [hunk 1]
```diff
@@ -194,4 +194,104 @@ definition {
...
+				siteId = "${siteId}",
+				title = "WC WebContent Title");
+		}
+
+		task ("When with curl I request getStructuredContentByVersion with restrictFields equal all fields except title field") {
+			var structuredContentId = HeadlessWebcontentAPI.getWebContentIdFromResponse(responseToParse = "${responseFromCreate}");
+
+			var response = HeadlessWebcontentAPI.versionStructureContent(
+				parameter = "restrictFields",
+				parameterValue = "actions,availableLanguages,contentFields,contentStructureId,creator,customFields,dateCreated,dateModified,datePublished,description,externalReferenceCode,friendlyUrlPath,id,key,keywords,numberOfComments,priority,relatedContents,renderedContents,siteId,subscribed,taxonomyCategoryBriefs,uuid,version",
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessAdminContentAPI/SupportContentVersioning/HeadlessAdminContentSupprotDifferentParameters.testcase:283]: Cool, I'm just wondering about the long term stability of the test. Will the list of fields be changed in the future? New ones added, or existing ones removed?

**Review by john-co — APPROVED** [review]
Looks good to me! :sunglasses:

Only 1 comment to consider.

**Comment by john-co** [comment]
ci:forward

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessAdminContentAPI/SupportContentVersioning/HeadlessAdminContentSupprotDifferentParameters.testcase (hunk 1 above)
Comment (john-co) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessAdminContentAPI/SupportContentVersioning/HeadlessAdminContentSupprotDifferentParameters.testcase:283]: cc/ @magjed4289

**Comment by john-co** [comment]
Please update test type and test status in the [test map](https://docs.google.com/spreadsheets/d/1_liLRC1XHBydH_mfeeDifgKxBWN3kfRtX7uqbKgJ72k/edit#gid=2145200593) with the completed tests

**Pull request closed without merging**